customisation/glossaries.md

```diff
@@ -5,3 +5,27 @@ layout: coming_soon
 title: Glossaries
 description: Terminology lists for training and customising machine translation
 ---
+#Glossaries
+
+Glossaries, i.e. lists of terms in one language aligned to their translation in one or more languages, can be added to machine translation systems.
+When a glossary term is matched in the source sentence, its translation is enforced in the output.
+
+## How glossaries are leveraged by MT systems
+Glossaries are leveraged at translation time, and no training is required.
+Adding a glossary to a commercial MT provider can increase the terminology accuracy of the output.
+However, the insertion of a term in the output happens regardless of the context.
+The terms included in the glossary should therefore be unambiguous, meaning that their translation should not be heavily dependent on the context.
+For example, "respiratory syndrome" is a good candidate to appear in a glossary, whereas the translation of the word "spread" may differ when it is used as a verb or a noun, or when the term occurs in the medical or in the financial domain.
+
+
+Several MT providers offer the possibility to enhance their output using a glossary.
+The way in which the glossary is leveraged may differ in terms of source term matching and target term insertion.
+In some cases the source term is matched in the source text only if it occurs with the same morphology and ortography as in the glossary.
+Some providers match the source term regardless of the morphology and ortography of its occurrence in the source sentence.
+
+
+The target term can be inserted either with the same form as in the glossary, or by adapting the term to the rest of the sentence and viceversa, for example ensuring a correct agreement between the subject and the verb.
```

Comment:
What do you think of adding a `Format` section before this paragraph? We can omit the following title and make this paragraph and the next part of the same section.

Reply:
Agreed, the article needed some reorganization. I'm not 100% sure of the name "Format". When I think about the format of a glossary I think of tsv, tbx, csv and so on :) I've chosen "Glossary term forms". Let me know if you're ok with this!

Reply:
Yeah, you are absolutely right.

Reply:
I suggest the same as for matching.

> Strict exact insertion is a problem because ...
>
> In some machine translation systems, term insertion can handle common variants, according to the context.
> - Morphological agreement
> - Uppercase and lowercase
> ...